src/commands/importWebApp/importWebApp.ts

```diff
@@ -0,0 +1,330 @@
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApiContract, BackendContract, BackendCredentialsContract, OperationCollection, OperationContract } from "azure-arm-apimanagement/lib/models";
+import WebSiteManagementClient from "azure-arm-website";
+import { Site, WebAppCollection } from "azure-arm-website/lib/models";
+import { ProgressLocation, window } from "vscode";
+import xml = require("xml");
+import { IOpenApiImportObject, OpenApiParser } from "../../../extension.bundle";
+import { IWebAppContract } from "../../azure/webApp/contracts";
+import * as Constants from "../../constants";
+import { ApisTreeItem } from "../../explorer/ApisTreeItem";
+import { ApiTreeItem } from "../../explorer/ApiTreeItem";
+import { ServiceTreeItem } from "../../explorer/ServiceTreeItem";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { apiUtil } from "../../utils/apiUtil";
+import { azureClientUtil } from "../../utils/azureClientUtil";
+import { processError } from "../../utils/errorUtil";
+import { nonNullValue } from "../../utils/nonNull";
+import { requestUtil } from "../../utils/requestUtil";
+
+export async function importWebAppToApi(node?: ApiTreeItem): Promise<void> {
+    if (!node) {
+        node = <ApiTreeItem>await ext.tree.showTreeItemPicker(ApiTreeItem.contextValue);
+    }
+
+    ext.outputChannel.show();
+    ext.outputChannel.appendLine(localize("importWebApp", "Import Web App started..."));
+
+    // tslint:disable: no-non-null-assertion
+    ext.outputChannel.appendLine(localize("importWebApp", "Getting Web App to import..."));
+    const pickedWebApp: Site = await getPickedWebApp(node, webAppKind.webApp);
+    const webAppResourceGroup = nonNullValue(pickedWebApp.resourceGroup);
+    const webAppName = nonNullValue(pickedWebApp.name);
+    const webConfigbaseUrl = getWebConfigbaseUrl(node!.root.environment.resourceManagerEndpointUrl, node!.root.subscriptionId, webAppResourceGroup, webAppName);
+    ext.outputChannel.appendLine(localize("importWebApp", "Getting picked Web App's config..."));
+    const webAppConfigStr: string = await requestUtil(webConfigbaseUrl, node.root.credentials, "GET");
+
+    // tslint:disable: no-unsafe-any
+    const webAppConfig: IWebAppContract = JSON.parse(webAppConfigStr);
+    if (webAppConfig.properties.apiDefinition && webAppConfig.properties.apiDefinition.url) {
+        ext.outputChannel.appendLine(localize("importWebApp", "Importing Web App from swagger object..."));
+        // tslint:disable-next-line: no-non-null-assertion
+        await importFromSwagger(webAppConfig, webAppName, node!.root.apiName, node!);
+    } else {
+        ext.outputChannel.appendLine(localize("importWebApp", "Can't find swagger object to import for this web app..."));
```

Comment:
API Definition not specified for webapp instead of "API Definition not specified for webapp"